src/annotation-engine/index.tsx

```diff
@@ -5,11 +5,14 @@ import Container from './style/container';
 import Image from './style/image';
 import useAnnotationEngine from './use-annotation-engine';
 
+export type DrawingEvent = 'mousemove' | 'drag';
```

Comment:
Ce qui peut être intéréssant pour ne pas laisser de "strings volants", c'est de passer par un `enum`.
A voir si dans ce cas ça te semble apporter quelque chose ou non.
```suggestion
export enum DrawingEvent {
    MOUSEMOVE = 'mousemove';
    DRAG = 'drag';
}
```
Ça permet, dans ta valeur par défaut plus bas, de mettre :
```ts
    drawingEvent = DrawingEvent.DRAG,
```

Ça aide à savoir explicitement les valeurs que tu peux passer, et tu peux faire des comparaisons avec plutôt qu'avec des strings (et, ça ne devrait pas être le cas ici, mais si tu changes un moment la valeur dans ton enum, tu n'as pas besoin de la modifier partout).

Reply:
Et vu que tu as besoin de ce type dans différents fichiers, ça vaudrait peut-être le coup de l'exporter dans un fichier `const.ts`